extensions/narayana-jta/deployment/src/main/java/io/quarkus/narayana/jta/deployment/NarayanaJtaLogFilterBuildStep.java

```diff
@@ -8,6 +8,5 @@
 
     @BuildStep
     void setupLogFilters(BuildProducer<LogCleanupFilterBuildItem> filters) {
-        filters.produce(new LogCleanupFilterBuildItem("com.arjuna.ats.arjuna", "ARJUNA012170"));
```

Comment:
Can you remove the method altogether in this case?

Reply:
No problem!